src/run.rs

```diff
@@ -448,6 +449,8 @@ fn touch_image(state: &mut State, image_id: &str, verbose: bool) -> io::Result<(
     // Get the current timestamp.
     match SystemTime::now().duration_since(UNIX_EPOCH) {
         Ok(duration) => {
+            let is_new_image = !state.images.contains_key(image_id);
```

Comment:
Can we use the result of `state.images.insert(...)` below (e.g., `Ok(state.images.insert(...).is_none())`) instead of this?

Reply:
Done